guides/setup-hassio-on-raspberrypi3-and-docker.md

```diff
@@ -0,0 +1,79 @@
+# Setup HASSIO on Raspberrypi3 with Docker
```

Comment:
A rendered version can be seen [here](https://github.com/Seeded-domotic/docs/blob/docs/how-to-setup-hassio-on-rasp3/guides/setup-hassio-on-raspberrypi3-and-docker.md)